server/api/users.js

```diff
@@ -3,6 +3,7 @@ const router = express.Router()
 const { User, Session, Cart, CartItem } = require('../db')
```

Comment:
I think this whole route should just be under /api/user, right? I don't think there needs to be a second route for users plural.

Reply:
Yes, I agree, will fix this as well